indexers/vector/PysparnnIndexer/__init__.py

```diff
@@ -27,20 +26,35 @@ def __init__(self,
                  prefix_filename: str = 'pysparnn_index',
                  *args, **kwargs):
         super().__init__(*args, **kwargs)
-
-        self.index = {}
         self.metric = self._assign_distance_class(metric)
         self.k_clusters = k_clusters
         self.num_indexes = num_indexes
         self.multi_cluster_index = None
         self.index_filename = prefix_filename + '_index.npz'
         self.indices_filename = prefix_filename + '_indices.npy'
 
+    def get_add_handler(self):
+        pass
+
+    def get_create_handler(self):
+        pass
+
+    def get_write_handler(self):
+        pass
+
+    def get_query_handler(self):
+        pass
+
     def post_init(self):
         import os
         super().post_init()
+        self.embeddings = []
```

Comment:
Remove `self.embeddings and `self.indices`. They are not used for anything. Leftovers from some tests

Reply:
Done